configure.host

```diff
@@ -7,6 +7,7 @@
 # Most of the time we can define all the variables all at once...
 case "${host}" in
   aarch64*-*-*)
+  	HAVE_EXT_VECTOR=1
```

Comment:
Do we really need this given the change in configure.ac that does the same thing?